pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.black]
 # line-length = 100
-target-version = ['py36', 'py37', 'py38', 'py39']
+target-version = ['py36', 'py37', 'py38', 'py39', 'py310']
```

Comment:
shouldnt we remove py36 if we dont test against it anymore

Reply:
from here an setup. cfg?

Reply:
Done!